betka.sh

```diff
@@ -86,6 +87,35 @@ function pull_cwt_image() {
   fi
 }
 
+function get_downstream_name_and_branch() {
+  local cwt_output
+  if [[ x"${OS}" == "xfedora" ]]; then
+    command="cwt"
+  else
+    command="rhcwt"
+  fi
+  cwt_output="${command}_output"
+  # Run CWT tool in order to get downstream name for specific version
+  if ! docker run -it --rm "${CWT_DOCKER_IMAGE}" "${command}" \
+      --config="${DOWNSTREAM_BRANCH}".yaml utils listupstream > "${cwt_output}"; then
```

Comment:
Will `--config="${DOWNSTREAM_BRANCH}".yaml` work for rhcwt? rhcw has rhel7.yaml and rhel8.yaml configs, but the branches are eg. `rhscl-3.6-rhel-7` and `rhel-8.3.0`

Reply:
Hopefully, I have addressed this issue by function `convert_branch_to_cwt_tool`